2주차/1758.py

```diff
@@ -0,0 +1,13 @@
+N = int(input())
+tip= [int(input()) for _ in range(N)]
+tip.sort(reverse=True)
+realtip=0
+total=0
+for i in range(N):
+    realtip=tip[i]-(i+1-1)
+    if realtip <0 :
+        continue
+    total+=realtip
```

Comment:
```suggestion
    if realtip >= 0 :
        total+=realtip
```
이렇게 써주는게 좀 더 읽기 편할 것 같네요